### ISSUE TITLE
Modal dialog a11y

### ISSUE THREAD

**Issue by LJWatson** [w3c/webex#10]
When the host has not joined a meeting, a dialog opens on top of other content. This dialog has the following a11y problems:
* It is not identified as a dialog to screen reader users;
* The title of the dialog is not communicated to screen reader users;
* The dialog does not receive keyboard focus when it opens;
* The dialog does not constrain keyboard focus inside it whilst it's open.

To resolve these problems:
* Set `role="dialog"` on the dialog container;
* Set `aria-modal="true"` on the dialog container;
* Set `aria-labelledby` on the dialog and point it to the dialogue message;
* Set keyboard focus to the `ok` button when the dialog opens;
* Confine keyboard focus inside the dialog whilst it's open;
* Prevent interaction with the underlying content whilst the dialog is open.

A simplified example of the existing code:
```
<div class="dialog-body" `role="dialog"` `aria-modal="true"` `aria-labelledby="p-message"`>
...
<div class="primary-message">
<p `id="p-message"` class="p-message">The host has not yet joined the meeting.</p>
</div>
...
<footer class="dlgfooter dlg-center" style="text-align: center; display: block;">
<button class="button medium JBH" data-monitor-id="pb:meeting:JBH-notify-ok">OK</button>
</footer>

</div>
```

Further information and examples available in the [ARIA 1.1 APG](https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal)

**Comment by LJWatson** [comment]
The same a11y issues apply to the "Meeting in progress" dialog, as well as to the "Getting started", "Video connection", and "Audio connection" dialogs.

Note: when one modal opens over the top of another (as the "Host has not yet..." modal opens over the "Getting started" modal, which in turn opens over the "Meeting in progress" modal, which in turn opens over the original page), it should only be possible to interact with the uppermost dialog.
